src/Docker.DotNet/DockerClient.cs

```diff
@@ -388,8 +388,12 @@ internal HttpRequestMessage PrepareRequest(HttpMethod method, string path, IQuer
         public void Dispose()
         {
             Configuration.Dispose();
+            if (_client != null)
+            {
+                _client.Dispose();
+            }
```

Comment:
Actually, the `if (_client != null)` appears superfluous because `_client` is assigned in the constructor and never cleared.